src/Quibill.Domain/Models/RecurringTransaction.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Quibill.Domain.Models
+{
+    class RecurringTransaction : ITransaction
+    {
+        public int TransactionId { get; }
+        public float TransactionAmount { get; }
+        public string TransactionType { get; }
+        public DateTime AddDate { get; }
+
+        public int TransactionRecurrenceDay { get; }
+
+
+        //Validates that the user doesn't select a transaction date outside the range of the month.
```

Comment:
@tloltman This validation only checks for the current month. I have to re-structure it to check the month that the user chooses. In what format will this be sent over from ionic when the user chooses a day on the calendar?